assets/prometheus-k8s/rules.yaml

```diff
@@ -304,6 +314,16 @@ spec:
       for: 15m
       labels:
         severity: critical
+    - alert: ClusterMonitoringOperatorDown
```

Comment:
Where is the jsonnet definition for this alert? I only see `ClusterMonitoringOperatorErrors` defined below.

Reply:
From here: https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/alerts/absent_alerts.libsonnet